src/scripts/components/main/navigation/navigation.tsx

```diff
@@ -34,6 +40,16 @@ const Navigation: FunctionComponent = () => {
         icon={StoryIcon}
         hideLabelOnMobile
       />
+      {isFilterActive && (
```

Comment:
I would suggest to rename `isFilterActive` to `isTagFilterActive` or you could also just say `selectedTags.length && `. This makes it easier to understand when the component should be visible.